log4j-core/src/main/java/org/apache/logging/log4j/core/net/SmtpManager.java

```diff
@@ -166,9 +172,7 @@ private static String createManagerName(
         sb.append(isDebug ? ":debug:" : "::");
         sb.append(filterName);
 
-        final String hash = NameUtil.md5(sb.toString());
-        return "SMTP:" + hash;
-
+        return "SMTP:" + sb.toString();
```

Comment:
Given `password` is in `sb`, this is not a good idea. Would you mind reverting this line, please?

Reply:
Hello @vy,
thanks for your feedback.
I've removed password from creation of smtp manager name, as it is irrelevant for creating a unique per-configuration name/key (cannot have same user name with different password on same host).
Greetings,
Markus